data_collection/gazette/spiders/ce_juazeiro_do_norte.py

```diff
@@ -0,0 +1,62 @@
+import datetime as dt
+
+import scrapy
+import re
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class CeJuazeiroDoNorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "2307304"
+    name = "ce_juazeiro_do_norte"
+    allowed_domains = ["juazeiro.ce.gov.br"]
+    start_date = None
+
+    def __init__(
```

Comment:
**suggestion**: The same behavior can be accomplished just providing two class arguments: `start_date` as `BASE_URL`. Usually we only add a `__init__` for a spider if we need do something more complex than that. All the logging is not that useful as well.